README.adoc

```diff
@@ -155,7 +157,14 @@ Revisit the `SystemClient` interface from the previous section:
 ----
 include::finish/src/main/java/io/openliberty/guides/inventory/client/SystemClient.java[tags=annotations]
 ----
-The `@Dependent` and `@RegisterRestClient` annotations imply that the interface is manageable through Context and Dependency Injection (CDI).
+The `@Dependent` annotation tells the service that the scope of the interface depends on the class that it is injected into.
+By default, interfaces have a scope of `@Dependent` unless there is another scope defined on the interface.
+
+The `@RegisterRestClient` annotation registers the interface as a RESTful client. The runtime create a CDI bean for every interface that is annotated with the `@RegisterRestClient` annotation.
+
+The `@Dependent` and `@RegisterRestClient` annotations imply that the interface is manageable through CDI. You must have them in order to inject the client by CDI.
+
+CDI also requires an CDI managed object to inject the `SystemClient` interface into. Since the `InventoryManager` class is another object that is managed by CDI, add the client here.
```

Comment:
Articles before acronyms can be weird. In this case, it should be "a CDI" because we pronounce "C" like "see."

It's also not ideal to end a sentence with a preposition. So how about...
"CDI also requires a CDI managed object. You can inject the `SystemClient` interface into the object."